src/app/frontend/logs/stateconfig.js

```diff
@@ -27,16 +27,10 @@ import {stateName} from './state';
  */
 export default function stateConfig($stateProvider) {
   $stateProvider.state(stateName, {
-    url: `${appendDetailParamsToUrl('/log')}/:container`,
-    params: {
-      'container': {
```

Comment:
No you have to rewrite that to match `resourceType` param. Logs page won't work correctly without thtat.